R/derive_merged.R

```diff
@@ -138,6 +147,7 @@
 #' @examples
 #' library(admiral.test)
 #' library(dplyr, warn.conflicts = FALSE)
+#' library(tibble)
 #' data("admiral_vs")
 #' data("admiral_dm")
 #'
```

Comment:
Could we make sure that @StefanThoma and @zdz2101 work to remove `admiral.test` from examples is not overwritten by this PR.  Unless this one is essential

Reply:
Git ensures this. The issue which is merged last should get a merge conflict.